authorization/src/main/java/com/terabite/chat/WebSocketConfig.java

```diff
@@ -28,6 +35,29 @@ public void configureMessageBroker(MessageBrokerRegistry registry) {
         registry.setUserDestinationPrefix(MESSAGE_PREFIX + "/user");
     }
 
+    // @Override
+    // public void configureClientInboundChannel(ChannelRegistration registration){
+    //     registration.interceptors(new ChannelInterceptor() {
+    //         @Override
+    //         public Message<?> preSend(Message<?> message, MessageChannel channel) {
+    //             StompHeaderAccessor accessor = MessageHeaderAccessor.getAccessor(message, StompHeaderAccessor.class);
+
+    //             if(StompCommand.CONNECT.equals(accessor.getCommand())){
+    //                 System.out.println("Connect ");
+    //             } else if(StompCommand.SUBSCRIBE.equals(accessor.getCommand())){
+    //                 System.out.println("Subscribe ");
+    //                 System.out.println(String.format("\nMessage: %s\n Channel: %s\n", message, channel));
+    //             } else if(StompCommand.SEND.equals(accessor.getCommand())){
+    //                 System.out.println("Send message " );
+    //             } else if(StompCommand.DISCONNECT.equals(accessor.getCommand())){
+    //                 System.out.println("Exit ");
+    //             } else {
+    //             }
+    //             return message;
+    //         }
+    //     });
+    // }
+
```

Comment:
Can we remove this if we are not going to add this back in, or is there another reason for keeping it here?

Reply:
No reason for it to still be there, removed.